packages/web/src/pages/upload-page/components/TrackModalArray.tsx

```diff
@@ -9,7 +9,7 @@ import styles from './TrackModalArray.module.css'
 export const TrackModalArray = () => {
   return (
     <div className={styles.root}>
-      <ReleaseDateModalForm />
+      <ReleaseDateField />
```

Comment:
will the rest of these be renamed or kept as "form"? I see how ReleaseDate is basically a fancy field, but the others all have multiple fields in the modal

Reply:
yeah that's exactly what im thinking, like at the end of the day these are fields that happen to render a contextual-menu that happens to render a form :D